subprojects/packagefiles/openblas/gen_install_headers.py

```diff
@@ -101,3 +102,4 @@ def main():
 
 if __name__ == "__main__":
     main()
+
```

Comment:
Hmm?

Reply:
Normally github's view of this would have indicated via some kind of red circle when the previous version didn't have a trailing end of line character, I thought. Meanwhile, the actual issue here is this adds a blank line. files ending with `main()` are a problem and files ending with `main()\n` are correct.

In fact, if I checkout the file locally, it doesn't appear to be missing the trailing end of line character. But if I checkout your PR, the file ends with `main()\n\n`, which is... surely wrong?